pkg/occlient/occlient.go

```diff
@@ -24,20 +26,20 @@ import (
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
 
+	scv1beta1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
 	servicecatalogclienset "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset/typed/servicecatalog/v1beta1"
+	appsv1 "github.com/openshift/api/apps/v1"
+	buildv1 "github.com/openshift/api/build/v1"
+	imagev1 "github.com/openshift/api/image/v1"
```

Comment:
why have you grouped everything together? 
api objects an clients were separated to make logical groups that are easier to search in

Reply:
it wasn't me :( 
it was my stupid IDE and auto go imports